.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = E402,W504
+exclude = .git,__pycache__,doc/source/conf.py,build,dist,versioneer.py,caproto/_headers.py,caproto/tests/test_pyepics_compat.py,caproto/_version.py
```

Comment:
`caproto/_headers.py,caproto/tests/test_pyepics_compat.py,caproto/_version.py` are not relevant. Please remove them.